pyproject.toml

```diff
@@ -1,3 +1,36 @@
+[tool.poetry]
+name = "athena-federation"
+version = "0.1.2"
+description = "Unofficial Python SDK for Athena Federation"
+authors = ["Damon P. Cortesi <DamonPCortesi@gmail.com>", "Ernest Prabhakar <ernest@quiltdata.io>"]
+license = "APACHE"
+readme = "README.md"
+homepage = "https://github.com/quiltdata/athena-federation-python-sdk"
+repository = "https://github.com/quiltdata/athena-federation-python-sdk"
+documentation = "https://github.com/quiltdata/athena-federation-python-sdk"
```

Comment:
Should we change these back to `dacort` repo?

Reply:
```suggestion
homepage = "https://github.com/dacort/athena-federation-python-sdk"
repository = "https://github.com/dacort/athena-federation-python-sdk"
documentation = "https://github.com/dacort/athena-federation-python-sdk"
```